.github/workflows/ci.yml

```diff
@@ -82,7 +82,14 @@ jobs:
       - name: 'Lint YAML for gotchas with yamllint'
         # Use '# yamllint disable-line rule:foo' to suppress.
         run: yamllint -c docs/.yamllint docs docs/.yamllint
-      - run: ./tensorboard/tools/docs_list_format_test.sh
+      - name: 'Install nbfmt.py for checking notebook formatting'
+        run: |
+          nbfmt_version="e9d8ddf8c0f1160d0eae654f30c028d2b0287538"
+          wget "https://raw.githubusercontent.com/tensorflow/docs/${nbfmt_version}/tools/nbfmt.py"
+          pip install absl-py
+          chmod +x ./nbfmt.py
+      - name: 'Check Colab notebooks for formatting'
+        run: git ls-files -z '*.ipynb' | xargs -0 ./nbfmt.py --test
```

Comment:
like the preventative work here to avoid quoted fns

Reply:
Yep! I generally try to be funnynames-safe when it’s not too
inconvenient, and in most cases it’s easy enough.